File: javascript_codes/arrayMedian.js
var findMedianSortedArrays = function(nums1, nums2) {
  let newArray = nums1.concat(nums2); // Combine the arrays
        newArray.sort((a, b) => a - b); // Sort the combined array in ascending order
        let arrayLength = newArray.length;

        if (arrayLength % 2 === 1) {
            let v1 = Math.floor(arrayLength / 2);
            return newArray[v1];
        } else {
            let v2 = Math.floor(arrayLength / 2);
            let sum1 = (newArray[v2 - 1] + newArray[v2]) / 2.0;
            return sum1;
        }  
};

File: python_codes/array_median.py
"""
Given two sorted arrays nums1 and nums2 of size m and n respectively, 
return the median of the two sorted arrays.
The overall run time complexity should be O(log (m+n)).
"""

def median_of_array(a1,a2):
	new_array = a1 
	new_array.extend(a2)
	new_array.sort()
	array_length = len(new_array)
	if array_length % 2 == 1:
		v1 = int(array_length/2) 
		return new_array[v1]
	else:
		v2 = int(array_length/2) 
		sum1 = float((new_array[v2-1] + new_array[v2])/2)
		return sum1


print(median_of_array([1,3],[2]))